packages/flutter_test/lib/src/widget_tester.dart

```diff
@@ -566,6 +566,80 @@ class WidgetTester extends WidgetController implements HitTestDispatcher, Ticker
     }).then<int>((_) => count);
   }
 
+  /// Triggers multiple frames for `duration` amount of time or until there are
+  /// no longer any frames scheduled, whichever is earlier.
+  ///
+  /// This does not promise settle of the widget, and there maybe remaining
```

Comment:
Widgets don't settle - maybe we can just say `This does not promise to exhaust all pending frames`?